databricks/koalas/frame.py

```diff
@@ -8331,6 +8331,106 @@ def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000):
 
         return DataFrame(internal) if not result_as_series else DataFrame(internal).T[key]
 
+    def query(self, expr, inplace=False):
+        """
+        Query the columns of a DataFrame with a boolean expression.
+
+        .. note::
+            * MultiIndex columns are not supported.
+            * Using variables in the environment with `@` is not supported.
+            * Internal columns that starting with __. are able to access,
+              however, they are not supposed to be accessed.
+            * This delegates to Spark SQL so the syntax follows Spark SQL
+            * If you want to use MultiIndex columns, or variables in the environment,
```

Comment:
> If you want to use MultiIndex columns

I don't think pandas supports this, right? You can just describe "if you want the exactly same syntax with pandas' you can work around by using ...`